bindings/python/pymongoarrow/types.py

```diff
@@ -266,6 +266,9 @@ def get_numpy_type(type):
     _atypes.is_boolean: _BsonArrowTypes.bool,
     _atypes.is_struct: _BsonArrowTypes.document,
     _atypes.is_list: _BsonArrowTypes.array,
+    _atypes.is_large_binary: _BsonArrowTypes.binary,
```

Comment:
Should we have a specific test for this type?

Reply:
I planned to leave the binary type alone. I'm glad you pointed out that line. It's the only place that it appears. if you'd like to add it, let's discuss. Otherwise, I'll delete this line.

Reply:
I'd say remove it if we're not going to test it.

Reply:
Done.